collections/DOrville/MS_DOrville_141.xml

```diff
@@ -37,45 +37,87 @@
                      <idno type="SCN">17019</idno>
                   </altIdentifier>
                </msIdentifier>
+               <head>Glosses on Sallust; Jean Lebègue. France, 15th century, second half</head>
                <msContents>
                   <msItem class="#commentarii" n="1" xml:id="MS_DOrville_141-item1">
-                     <title key="work_12051" type="desc">Glosses on Sallust, Catilina and Jugurtha</title>
-                     <bibl type="commentedOn">
-                        <author key="person_104162705">Sallust</author>
-                        <title>Catilina</title>
-                     </bibl>
-                     <bibl type="commentedOn">
-                        <author key="person_104162705">Sallust</author>
-                        <title>Jugurtha</title>
-                     </bibl>
-                     <textLang mainLang="la">Latin</textLang>
+                      <title key="work_12051" type="desc">Glosses on Sallust, Catilina and Jugurtha</title>
+                     <rubric>Incipiunt Glose supra Catalinarium et Iugurtinum Salustij felicter</rubric>
+                     <msItem>
+                        <locus>(fols. 1r-18v)</locus>
+                        <incipit>Salustius uti mos erat</incipit>
+                        <bibl><ref target="http://catalogustranslationum.org">Catalogus Translationum et Commentariorum</ref>, I.10a (vol. X, pp. 380-3, printing the accessus in full, and with further bibliography). </bibl>
```

Comment:
Personally I think we should reserve links for long-term resources available online, whereas this is mostly a marketing website.

Reply:
that seems overly harsh to me! It does makes the text of vols. 1-10 freely available after all...